portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,4 +42,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json");
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Javadoc would be a good place to document the expected overall behaviour of the request handler.

Reply:
Added Javadoc